include/spatio_temporal_voxel_layer/spatio_temporal_voxel_layer.hpp

```diff
@@ -126,21 +131,32 @@ class SpatioTemporalVoxelLayer : public costmap_2d::CostmapLayer
   bool AddStaticObservations(const observation::MeasurementReading& obs);
   bool RemoveStaticObservations(void);
 
+  // Dynamic reconfigure
+  void DynamicReconfigureCallback(dynamicReconfigureType &config, uint32_t level);
+  dynamicReconfigureServerType* _dynamic_reconfigure_server;
+
+  bool BufferEnablerCallback( std_srvs::SetBool::Request & request,    \
+                              std_srvs::SetBool::Response & response,  \
+                              boost::shared_ptr<buffer::MeasurementBuffer>& buffer, \
+                              boost::shared_ptr<message_filters::SubscriberBase>& subcriber);
+
   laser_geometry::LaserProjection                                  _laser_projector;
   std::vector<boost::shared_ptr<message_filters::SubscriberBase> > _observation_subscribers;
   std::vector<boost::shared_ptr<tf::MessageFilterBase> >           _observation_notifiers;
   std::vector<boost::shared_ptr<buffer::MeasurementBuffer> >       _observation_buffers;
   std::vector<boost::shared_ptr<buffer::MeasurementBuffer> >       _marking_buffers;
   std::vector<boost::shared_ptr<buffer::MeasurementBuffer> >       _clearing_buffers;
+  //std::vector<boost::shared_ptr<ros::ServiceServer> >              _buffer_enabler_servers;
+  std::vector<ros::ServiceServer*>                                 _buffer_enabler_servers;
```

Comment:
If you're switching from boost pointers, remember to delete them in the destructor